### DISCUSSION TITLE
吐槽《雪中》的文案 | YuZhangWang的领域

### DISCUSSION THREAD

**Discussion by giscus** [YuZhangWang/YuZhangWang.github.io#13] [Announcements]
# 吐槽《雪中》的文案 | YuZhangWang的领域

Hi,各位小伙伴大家好,我是YuZhangWang 前不久,雪中悍刀行第一季已经完结了,在庆余年中的父愁者联盟，陈道明(加图片)等老戏骨们的演技，让我对雪中悍刀行这部电视剧有些许的期待。带着对胡军、杜玉明、王同辉、李解(加图片)这些老师演技的信任,怀着我的期待,追完了这部剧 但是在这一部剧中,我陷入了失望当中,那么在一起视频当中，让我作为一个非专业的解说者,来聊一聊这一部剧让我比较难受的一些槽点

https://www.yuzhang.wang/2022/08/05/119-review-draft/

**Comment by hjljy** [comment]
很久没看电视局了，实在是不敢恭维现在的很多电视剧

**Reply by YuZhangWang** [reply]
是的，现在很多都是借电视剧洗钱罢了，有时候运气不好，被宣发忽悠了。还是得找点靠谱得影视UP，听听人家推荐，再去看看好不好看。比如木鱼水心，迷影至下这些UP，都是不错的指向标。
